src/at_tools/execute_att.rs

```diff
@@ -87,7 +90,7 @@ pub async fn run_tools(
             let mut have_answer = false;
             for msg in tool_msg_and_maybe_more {
                 if let ContextEnum::ChatMessage(ref raw_msg) = msg {
-                    if (raw_msg.role == "tool" || raw_msg.role == "diff") && raw_msg.tool_call_id == t_call.id {
+                    if (raw_msg.role == "tool" || raw_msg.role == "diff" || raw_msg.role == "supercat" || raw_msg.role == "search") && raw_msg.tool_call_id == t_call.id {
```

Comment:
why we need cat and search roles?

Reply:
already solved 👌🏻. It was needed for completing content for this roles -- attaching which files were ignored after postprocessing, after what they became regular "tool" messages.